README.md

```diff
@@ -111,7 +109,6 @@ Ensure you have successfully installed all system package dependencies as descri
 ```
 $ git clone https://github.com/snowgem/snowgem.git
 $ cd snowgem/
-$ chmod +x zcutil/build-win.sh depends/config.guess depends/config.sub autogen.sh share/genbuild.sh src/leveldb/build_detect_platform  depends/Makefile depends/cargo-checksum.sh
```

Comment:
I don't have a windows system so I'm not 100% certain on this. Can anyone confirm that the changes to the repository allows build to be successful on Windows also? I tested on Linux without issue.

Reply:
I built Windows/Linux daemon on Windows 10 with Bash without any issue